arch/arm64/boot/dts/freescale/imx8mp-edm-g-wb.dts

```diff
@@ -300,6 +303,11 @@
 		pinctrl-names = "default";
 		#gpio-cells = <2>;
 		status = "okay";
+		/* below tags for gpio pins are updated to sapphire 2.3.4.2 */
```

Comment:
below tags for gpio pins are updated *with respect* to sapphire 2.3.4.2 *schematic*